apps/demos/Demos/Chat/Customization/jQuery/index.js

```diff
@@ -0,0 +1,70 @@
+$(() => {
+  const chat = $('#chat').dxChat({
+    height: 710,
+    items: messages,
+    user: currentUser,
+    dayHeaderFormat,
+    onMessageEntered({ component, message }) {
+      component.renderMessage(message);
+    },
+    messageTimestampFormat,
+  }).dxChat('instance');
+
+  $('#chat-disabled').dxCheckBox({
+    value: false,
+    text: 'Disable Chat',
+    onValueChanged(data) {
+      chat.option('disabled', data.value);
+    },
+  });
+
+  $('#show-day-headers').dxCheckBox({
+    value: true,
+    text: 'Show Day Headers',
+    onValueChanged(data) {
+      chat.option('showDayHeaders', data.value);
+    },
+  });
+
+  $('#show-message-timestamp').dxCheckBox({
+    value: true,
+    text: 'Show Message Timestamp',
+    onValueChanged(data) {
+      chat.option('showMessageTimestamp', data.value);
+    },
+  });
+
+  $('#show-avatar').dxCheckBox({
+    value: true,
+    text: 'Show Avatar',
+    onValueChanged(data) {
+      chat.option('showAvatar', data.value);
+    },
+  });
+
+  $('#show-user-name').dxCheckBox({
+    value: true,
+    text: 'Show User Name',
+    onValueChanged(data) {
+      chat.option('showUserName', data.value);
+    },
+  });
+
+  $('#day-headers-format').dxSelectBox({
+    items: dayHeaderFormat,
+    value: dayHeaderFormat[0],
```

Comment:
Kindly use the same approach for messageTimestampFormat. The approaches should be consistent

Reply:
https://github.com/DevExpress/DevExtreme/pull/28441/commits/7abeedba5198baaf75acd81fdb9e03e779b24d94